avocado/utils/linux_modules.py

```diff
@@ -103,9 +105,10 @@ def loaded_module_info(module_name):
              dependent on, list of dictionary of param name and type
     :rtype: dict
     """
-    l_raw = process.system_output('/sbin/lsmod')
+    encoding = locale.getpreferredencoding()
```

Comment:
we are still negotiating how to get the accurate default, so after https://github.com/avocado-framework/avocado/pull/2611 we might need to modify this.

Reply:
this change is not related to the commit message. It should be separated...